src/utils/logger.js

```diff
@@ -13,6 +13,7 @@ const LogLevel: LogLevelType = {
   OFF: JsLogger.OFF
 };
 
+// TODO sakal check why not using playkit-js logger
```

Comment:
In general the UI app was supposed to be playkit-js dependency free and only operate on the interface.
We tried but obviously didn't do the best-est of job, but I would like to keep it for sake of when we can define clear interface.
The real challenge here can be solved by the new bundler that can inject the logger as app wide dependency and then you avoid the duplication.
Note: you might say that the player interface has a logger, which is true and we can base upon this - open for discussion.

Reply:
ok, will use this logger in my plugins